katcp/inspecting_client.py

```diff
@@ -519,13 +519,38 @@ def handle_sensor_value(self):
 
     @tornado.gen.coroutine
     def inspect(self):
-        """Inspect device requests and sensors, update model"""
+        """Inspect device requests and sensors, update model
+
+        Returns
+        -------
+
+        Tornado future that resolves with:
+
+        model_changes : Nested AttrDict
```

Comment:
If there were neither request changes nor sensor changes, then the return value will resolve to `None`.
Suggest adding "or None".